pkg/defaults/deployment_defaults.go

```diff
@@ -23,8 +23,23 @@ import (
 
 	appsv1 "k8s.io/api/apps/v1"
 	"knative.dev/pkg/apis"
+)
+
+const (
+	// ServingGroupName is the group name for knative labels and annotations
+	ServingGroupName = "serving.knative.dev"
+
+	// ServingServiceLabelKey is the label key attached to a Route and Configuration indicating by
+	// which Service they are created.
+	ServingServiceLabelKey = ServingGroupName + "/service"
```

Comment:
Those don't seem to be used crosspackage, so probably lowecase?

Reply:
Done